backend/src/main/java/com/openelements/maven/initializer/backend/service/ProjectGeneratorService.java

```diff
@@ -156,6 +169,55 @@ private void generatePomFile(Path projectDir, ProjectRequestDTO request) {
     }
   }
 
+  private void addDefaultDependencies(PomEditor editor) {
+    var root = editor.root();
+    var depsTmp = editor.findChildElement(root, MavenPomElements.Elements.DEPENDENCIES);
+
+    if (depsTmp == null) {
+      depsTmp = editor.insertMavenElement(root, MavenPomElements.Elements.DEPENDENCIES);
+    }
+
+    final var deps = depsTmp;
+    DEFAULT_DEPENDENCIES.forEach(
+        ga -> {
+          String[] parts = ga.split(":", 2);
+          if (parts.length == 2) {
+            var depEl = editor.insertMavenElement(deps, MavenPomElements.Elements.DEPENDENCY);
+            editor.insertMavenElement(depEl, MavenPomElements.Elements.GROUP_ID, parts[0]);
+            editor.insertMavenElement(depEl, MavenPomElements.Elements.ARTIFACT_ID, parts[1]);
+            editor.insertMavenElement(depEl, MavenPomElements.Elements.SCOPE, "test");
+          }
+        });
+  }
+
+  private void addDefaultDependencyManagement(PomEditor editor) {
+    var root = editor.root();
+    var dm = editor.findChildElement(root, MavenPomElements.Elements.DEPENDENCY_MANAGEMENT);
+
+    if (dm == null) {
+      dm = editor.insertMavenElement(root, MavenPomElements.Elements.DEPENDENCY_MANAGEMENT);
+    }
+
+    var dmsTmp = editor.findChildElement(dm, MavenPomElements.Elements.DEPENDENCIES);
+    if (dmsTmp == null) {
+      dmsTmp = editor.insertMavenElement(dm, MavenPomElements.Elements.DEPENDENCIES);
+    }
+
+    final var dms = dmsTmp;
+    DEFAULT_DEPENDENCY_MANAGEMENT.forEach(
+        bom -> {
+          String[] parts = bom.split(":", 3);
```

Comment:
Because you are operating with spiltting, we should think about to introduce a domain object for dependencies, that would increase the code readibility